nightwatch.conf.js

```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
+
+module.exports = {
+    'src_folders': ['tests'],
+    'page_objects_path': ['pageobjects'],
+
+        'webdriver': {
```

Comment:
Just a minor quibble.  I'd fix the indentation here.  Might be a good idea to set your IDE to either auto-indent, or use a linter to show when columns are mis-aligned

Reply:
Set up a linter. I think it will help keep me consistent. Thanks Clint!